docs/kubernetes/getstarted.md

```diff
@@ -0,0 +1,604 @@
+
+**NOTE** that you can't give the same subnet to all of your nodes. Each node must be assigned a different /24 subnet that is part of the global /16 of the network.
+
+We will need an external access to the network to be able to ssh into our VMs.
+
+```
+$ ./tfuser generate --schema network.json network add-access --node qzuTJJVd5boi6Uyoco1WWnSgzTb7q8uN79AjBT9x9N3 \
+				      --subnet 173.30.4.0/24 --ip4 > wg.conf
+```
+
+We generated the wireguard, a secure and fast VPN, configuration file that we will use here after in this guide.
+
+Let's take a look at a graphical representation of our network
+
+```
+$ ./tfuser generate --schema network.json network graph
+```
+
+This will generate network.json.dot file. You can copy paste the content on a website like [graphiz](https://dreampuf.github.io/GraphvizOnline) to see the visual representation of the network we defined.
+
+![network graphical representation](network_dot_file.png)
+
+Now we have a nodes in a network definition ready to be provisioned. Let's use our seed to sign the transaction and ask for a 2 hours provision.
+
+```
+$ ./tfuser provision --schema network.json --duration 2 --seed user.seed \
+				      --node qzuTJJVd5boi6Uyoco1WWnSgzTb7q8uN79AjBT9x9N3 \
+				      --node FTothsg9ZuJubAEzZByEgQQUmkWM637x93YH1QSJM242 \
+				      --node 3NAkUYqm5iPRmNAnmLfjwdreqDssvsebj4uPUt9BxFPm
+Reservation for 48h0m0s send to node qzuTJJVd5boi6Uyoco1WWnSgzTb7q8uN79AjBT9x9N3
+Resource: /reservations/2406-1
+Reservation for 48h0m0s send to node FTothsg9ZuJubAEzZByEgQQUmkWM637x93YH1QSJM242
+Resource: /reservations/2407-1
+Reservation for 48h0m0s send to node 3NAkUYqm5iPRmNAnmLfjwdreqDssvsebj4uPUt9BxFPm
+Resource: /reservations/2408-1
+```
+
+you can check the status of the provision with the live command
+
+```
+
+$ ./tfuser live --seed user.seed --end 3000
+ID:2406-1 Type:   network expired at:31-Jan-2020 16:18:31state:     ok  network ID:
+ID:2407-1 Type:   network expired at:31-Jan-2020 16:18:31state:     ok  network ID:
+ID:2408-1 Type:   network expired at:31-Jan-2020 16:18:31state:     ok  network ID:
+
+```
+
+We will now use the wg.conf file previously generated to connect to our network with wireguard a secure and fast vpn.
+be sure to install wireguard and then simply run this command to create a network interface and join the defined network
+
+```
+$ wg-quick up ./wg.conf
+```
+
+We have setup and provision a network on the grid that we can join through wireguard.
+Let's now provision some Kubernetes VM on those nodes
+
+### Provision
+
+Let's provision a kubernetes VM on our first node and assign an IP allowed in the network `kubetest` defined here above.
+Replace `GITHUB_USERNAME` by your github username. We choose an IP for the VM that is part of the subnet we previously defined for the node. We can't choose 173.30.3.1 as it is the IP of the network ressource itself.
```

Comment:
Instead of using github username let's show how they can just give their public SSH key here. And as a note, example github username is possible.

Reply:
done